docs/serving/spec/knative-api-specification-1.0.md

```diff
@@ -148,6 +148,39 @@ Service object to reduce the cognitive overhead for application developers – t
 Service object should provide sufficient controls to cover most of application
 deployment scenarios (by frequency).
 
+## Extentions
+
+Extending the Knative resource model allows for custom semantics to be
+offered by implementions of the specification. Unless otherwise noted,
+implementations of this specification MAY define extensions but those
+extensions MUST NOT contradict the semantics defined within this specification.
+
+There are several ways in which implementations can extend the model:
+- Annotations and Labels
+  Allowing end users to include annotations or labels on the Knative resources
+  allows for them to indicate that they would like some additional semantics
+  applied to those resources. When defining annotations, or labels, it
```

Comment:
Reference https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions ?

In particular:

> Annotations have very different intended usage from labels. They are primarily generated and consumed by tooling and system extensions, or are used by end-users to engage non-standard behavior of components. For example, an annotation might be used to indicate that an instance of a resource expects additional handling by non-kubernetes controllers. Annotations may carry arbitrary payloads, including JSON documents. **Like labels, annotation keys can be prefixed with a governing domain (e.g. example.com/key-name). Unprefixed keys (e.g. key-name) are reserved for end-users. Third-party components must use prefixed keys.** Key prefixes under the "kubernetes.io" and "k8s.io" domains are reserved for use by the kubernetes project and must not be used by third-parties.

(emphasis mine)

Reply:
done